### PR TITLE
Bug 2043572: Deleting namespace store in UI secrets remains

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
Remove secret which was created automatically during
namespace creation and Keep the secret if its created
manually and assigned to namespace.

Signed-off-by: Timothy Asir Jeyasingh <tjeyasin@redhat.com>

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
@bipuladh , @GowthamShanmugam , Please review

**Comment by TimothyAsirJeyasing** [comment]
@bipuladh , @GowthamShanmugam , Please review

File: frontend/public/components/modals/delete-modal.tsx [hunk 1]
```diff
@@ -36,6 +61,20 @@ const DeleteModal = withHandlePromise((props: DeleteModalProps & HandlePromisePr
     props.handlePromise(k8sKill(kind, resource, {}, {}, json), () => {
       props.close();
 
+      if (resource.kind === 'NamespaceStore') {
```
Comment (GowthamShanmugam) [inline frontend/public/components/modals/delete-modal.tsx:64]: I think it needs to be deleted using owner ref

Comment (GowthamShanmugam) [reply frontend/public/components/modals/delete-modal.tsx:64]: i think this is migrated to odf-console.

cc: @rexagod

Comment (TimothyAsirJeyasing) [reply frontend/public/components/modals/delete-modal.tsx:64]: Here it deletes the secret only if the secret is created automatically

**Comment by SanjalKatiyar** [comment]
/close
this PR needs to go in diff repo...

**Pull request closed without merging**